frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -371,8 +371,6 @@ action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.save-and-edit-but
 action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.title=Discovered branch layout
 action.GitMachete.DiscoverAction.notification.title.cannot-get-current-repository-error=Can't get the current git repository
 action.GitMachete.DiscoverAction.notification.title.repository-discover-error=Repository discover error
-action.GitMachete.DiscoverAction.notification.title.write-file-error=Can't write discovered branch layout
```

Comment:
Meh, if an error happens on writing .git/machete, let's better throw a proper error for the user to report